appveyor.yml

```diff
@@ -92,11 +92,11 @@ install:
   - C:\Python35-x64\Scripts\pip.exe install -e .
 
   # point to tsc 3.2.1
```

Comment:
comment should match installed typescript (or remove version number from the comment).